### DISCUSSION TITLE
Python yield kullanımı nasıldır? yield nerede kullanılır? | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#342] [Site]
# Python yield kullanımı nasıldır? yield nerede kullanılır? | SonsuzUs Github

Python yield kullanımı

https://sonsuzus.github.io/posts/python-yield-kullanimi/

**Comment by sonsuzus** [comment]
Faktoriyel alan fonksiyon. (yield kullanarak)

```python
def fak():
    carpim,i = 1,1
    while True:
        yield carpim
        i +=1
        carpim *= i

for i in fak():
    if i >100:
        print(i)
        break
```

sonuç olarak 120 yazacaktır.
